www/js/lib/uiUtil.js

```diff
@@ -969,16 +982,20 @@ function warnAndOpenExternalLinkInNewTab (event, clickedAnchor, archive) {
         clickedAnchor.href = clickedAnchor.href.replace(clickedAnchor.origin, archive.source.replace(/\/$/, ''));
     }
     var target = clickedAnchor.target;
+    if (!target) {
+        target = '_blank';
+    }
+    if (params.hideExternalLinkWarning) {
+        window.open(clickedAnchor.href, target);
+        return;
+    }
     var message = translateUI.t('dialog-open-externalurl-message') || '<p>Do you want to open this external link?';
     if (!target || target === '_blank') {
```

Comment:
Because you moved `if (!target)` test to above, you can remove the `!target || ` part here.

Reply:
Removed in [d0150e5](https://github.com/kiwix/kiwix-js/pull/1236/commits/d0150e5d4a86d77c7cb4b456abd49a292d5c7f24).